src/dlgTriggerEditor.cpp

```diff
@@ -125,68 +119,56 @@ dlgTriggerEditor::dlgTriggerEditor( Host * pH )
     setStatusBar( statusBar );
     statusBar->show();
     mIsGrabKey = false;
-    QVBoxLayout * pVB1 = new QVBoxLayout(mainArea);
+
+    // Replacement code to use single splitter for all right widgets:
+    mpSystemMessageArea = new dlgSystemMessageArea(splitter_rightTopBottom);
+    mpTriggersMainArea = new dlgTriggersMainArea(splitter_rightTopBottom);
+    mpTimersMainArea = new dlgTimersMainArea(splitter_rightTopBottom);
+    mpAliasMainArea = new dlgAliasMainArea(splitter_rightTopBottom);
+    mpActionsMainArea = new dlgActionMainArea(splitter_rightTopBottom);
+    mpKeysMainArea = new dlgKeysMainArea(splitter_rightTopBottom);
+    mpVarsMainArea = new dlgVarsMainArea(splitter_rightTopBottom);
+    mpScriptsMainArea = new dlgScriptsMainArea(splitter_rightTopBottom);
+    mpSourceEditorArea = new dlgSourceEditorArea(splitter_rightTopBottom);
+    mpErrorConsole = new TConsole(mpHost,false,splitter_rightTopBottom);
+    mpErrorConsole->setMinimumHeight( 200 );
+
+    //Prevent any region being "squashed down" i.e. taken below its minimum size
+    //so that it "disappears"
+    splitter_rightTopBottom->setChildrenCollapsible(false);
```

Comment:
But in my comment right above https://github.com/Mudlet/Mudlet/pull/436#discussion_r108057886, I say that I do not want it to be tied to the small screen option.... ?